### ISSUE TITLE
Verbose Debug Information

### ISSUE THREAD

**Issue by cevek** [Microsoft/TypeScript#25724]
**TypeScript Version:**  3.0.0-dev.20180712

**Search Terms:**

**Code**

```ts
Verbose Debug Information: {"pos":426,"end":431} and {"pos":427,"end":621}
    at _loop_11 (/usr/local/lib/node_modules/typescript/lib/tsserver.js:103334:34)
    at /usr/local/lib/node_modules/typescript/lib/tsserver.js:103340:25
    at Array.map (native)
    at Object.getTextChangesFromChanges (/usr/local/lib/node_modules/typescript/lib/tsserver.js:103328:86)
    at ChangeTracker.getChanges (/usr/local/lib/node_modules/typescript/lib/tsserver.js:103305:45)
    at Function.ChangeTracker.with (/usr/local/lib/node_modules/typescript/lib/tsserver.js:102881:32)
    at Object.getCodeActions (/usr/local/lib/node_modules/typescript/lib/tsserver.js:105918:65)
    at /usr/local/lib/node_modules/typescript/lib/tsserver.js:103819:121
    at Object.flatMap (/usr/local/lib/node_modules/typescript/lib/tsserver.js:498:25)
    at Object.getFixes (/usr/local/lib/node_modules/typescript/lib/tsserver.js:103819:23)
    at /usr/local/lib/node_modules/typescript/lib/tsserver.js:111665:35
    at Object.flatMap (/usr/local/lib/node_modules/typescript/lib/tsserver.js:498:25)
    at Object.getCodeFixesAtPosition (/usr/local/lib/node_modules/typescript/lib/tsserver.js:111663:23)
    at IOSession.Session.getCodeFixes (/usr/local/lib/node_modules/typescript/lib/tsserver.js:119225:64)
    at Session.handlers.ts.createMapFromTemplate._a.(anonymous function) (/usr/local/lib/node_modules/typescript/lib/tsserver.js:118099:61)
    at /usr/local/lib/node_modules/typescript/lib/tsserver.js:119389:88
    at IOSession.Session.executeWithRequestId (/usr/local/lib/node_modules/typescript/lib/tsserver.js:119380:28)
    at IOSession.Session.executeCommand (/usr/local/lib/node_modules/typescript/lib/tsserver.js:119389:33)
    at IOSession.Session.onMessage (/usr/local/lib/node_modules/typescript/lib/tsserver.js:119409:35)
    at Interface.<anonymous> (/usr/local/lib/node_modules/typescript/lib/tsserver.js:120676:27)
```

**Expected behavior:**

**Actual behavior:**

**Playground Link:** 

**Related Issues:** 

tsserver log id: 3897983895

**Comment by sheetalkamat** [comment]
Looking at the logs and investigating it shows that this should be fixed by #28217
@cevek If this still repros with typescript@next please provide latest tsserver log and we can relook into it. Thanks.

**Comment by sheetalkamat** [comment]
Duplicate of #27700 which is fixed by #28217
